data/infra/resources/powershell_script.yaml

```diff
@@ -128,6 +96,14 @@ properties_list:
   required: false
   description_list:
   - markdown: The current working directory from which the command will be run.
+- property: default_env
```

Comment:
Is default_env valid here?

Reply:
This file appears to be pure garbage that got built by some errant AI. Some of the code it references comes from PR's merged in 2015. For now, you should nuke this file.